docs/installation_quickstart.rst

```diff
@@ -55,8 +55,12 @@ being picked up correctly and that the tracer will be able to connect to the Dat
 Note: ``--info`` Only reflects configurations made via environment variables, not those made in code.
 
 
-If ``ddtrace-run`` isn't suitable for your application then :py:func:`ddtrace.patch_all`
-can be used::
+When ``ddtrace-run`` cannot be used, a similar start-up behavior can be achieved
+with the import of ``ddtrace.auto``. This should normally be imported as the
```

Comment:
looking over this again, should we provide a code example? I know it seems weird given it is just `import ddtrace.auto`... but we show one for `patch_all()`.... wdyt? would that help clarify things at all?

Reply:
Yep I think that would help. Do you have something in mind that is generic enough that we can include here?